src/Footer.js

```diff
@@ -0,0 +1,9 @@
+const Footer = props => (
+  <ul className="footer">
+    {props.arr.map((eachItem, index) => (
+      <li key={index}>{eachItem}</li>
```

Comment:
Good job remembering the `key` property, many even professionals forget this! A note of caution though, try not to use the `index` of the array as the key, because (especially when you start dealing with state in arrays) the entry found at each index can change, but because the index doesn't change React won't re-render the entry in the array. The `key` should uniquely identify each element in an array.